frontend/src/views/Game.tsx

```diff
@@ -108,11 +112,22 @@ function GamePage() {
 
     // Get the result of promises and set the default code list.
     Promise.all(promises).then((result) => {
+      const codeMap = result[0];
+
+      // If previous code and language specified, save those as defaults
+      if (code) {
+        codeMap[language] = code;
+      }
```

Comment:
Yeah I saw the other comment, this is what I replied: 

> Might have misstated this but yeah as of now this is what it does (gets the latest submitted or ran code - although not using updateCode since I don't think we've made that a public endpoint yet). However unless we periodically sync or save multiple languages at a time, it has the issues mentioned above - but I agree about the local storage stuff, I think that's prob the best path moving forward in future PRs

So yeah in essence I agree that at some point we'll want to be able to save multiple languages - I think we'll probably want to use localstorage as you suggested in the other comment, that way everything's saved regardless of running/submitting (and also helps to keep the backend logic simpler)

In the meantime I'll add a card for this in the backlog